src/common/transformations/src/transformations/transpose_sinking/ts_reduction.cpp

```diff
@@ -82,16 +81,21 @@ TSReductionForward::TSReductionForward() {
                                                     {transpose_order_values.size()},
                                                     transpose_order_values);
 
-        auto new_const = Constant::create(reduction_axes->get_element_type(), reduction_axes->get_shape(), new_values);
-        auto new_reduction = reduction->clone_with_new_inputs({transpose->input_value(0), new_const});
-        auto new_transpose = transpose->clone_with_new_inputs({new_reduction, new_transpose_order});
+        auto new_const = Constant::create(reduction_axes->get_element_type(), {new_values.size()}, new_values);
```

Comment:
what is the reduction axes are not constant, should we have more smart transformation for its permutation?

Reply:
The current limitation that the axis ia a constant.
I created a ticket for the next release: 108225